homeassistant/core.py

```diff
@@ -1260,7 +1260,7 @@ class Service:
 
     def __init__(
         self,
-        func: Callable,
+        func: Callable[[ServiceCall], Any],
```

Comment:
Do we allow `Any` as return type?
Shouldn't it be `None | Awaitable[None]`

Reply:
```suggestion
        func: Callable[[ServiceCall], Awaitable[None] | None],
```

Reply:
I'm testing it at the moment, but in principle you're right. Although `Any` would also be possible, no value is ever returned from a service call.

Reply:
> Shouldn't it be `None | Awaitable[None]`

👍🏻 Changed in the last commit